SpatialGDK/Documentation/content/helper-scripts.md

```diff
@@ -1,8 +1,6 @@
-<%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
-
 # Helper scripts
 
-These scripts are located in your UnrealGDK repo under `SpatialGDK\Build\Scripts\`
+These scripts are located under `<GDKDir>\SpatialGDK\Build\Scripts\` where `<GDKDir>` is the path to were you have the GDK within your project(normally in the `Plugins` folder of your game).
```

Comment:
<ProjectRoot>/<GameRoot>/Plugins/UnrealGDK/

or maybe

The UnrealGDK repository which will be located in your Games Plugin folder. It must be called `UnrealGDK`.